postCompose/src/main/java/com/hamdan/forzenbook/post/compose/Post.kt

```diff
@@ -95,12 +104,19 @@ private fun TextPostContent(
                     focusRequester.requestFocus()
                 },
         ) {
-            PostTextField(
-                state.getText(),
-                focusRequester
-            ) {
-                onTextChange(it)
-            }
+            state.asTextOrNull()?.apply {
```

Comment:
The null or non null blocks do mostly the same thing. It doesn't make sense to elvis and duplicate

Reply:
I see what your saying, I think.